src/bitcoin/esplora_client.rs

```diff
@@ -10,17 +10,23 @@ use crate::{
     result::JsResult,
     types::{FullScanRequest, SyncRequest, Update},
 };
+use std::time::Duration;
```

Comment:
Just for the record, even if we are using `std` here, this seems to be ok for our current target!